From: Cole Robinson <crobinso@redhat.com>
Date: Thu, 23 Jun 2016 16:23:11 -0400
Subject: [PATCH] spec: Fix error in last backport

---
 libvirt.spec.in | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/libvirt.spec.in b/libvirt.spec.in
index c1453a9..3fa5fc4 100644
--- a/libvirt.spec.in
+++ b/libvirt.spec.in
@@ -344,6 +344,14 @@
 %endif
 
 
+# The RHEL-5 Xen package has some feature backports. This
+# flag is set to enable use of those special bits on RHEL-5
+%if 0%{?rhel} == 5
+    %define with_rhel5  1
+%else
+    %define with_rhel5  0
+%endif
+
 %if 0%{?fedora} >= 18 || 0%{?rhel} >= 7
     %define with_systemd_macros 1
 %else
